# goal
uint32 SIDEWAYS_GRASP=0
uint32 TOP_GRASP=1

geometry_msgs/PoseStamped pose
uint32 strategy
---
# result
bool success
---
# feedback
string current_state
string message
